recipe/meta.yaml

```diff
@@ -50,10 +50,15 @@ test:
     - hypothesis >=3.58
     - pytest >=6.0
     - pytest-xdist
+    - pytest-html
   imports:
     - pandas
+  source_files:
+    - pandas/tests
   commands:
     - pip check
+    - python -c "import pandas; pandas.test(['-k', 'test_aggregation', '--html="reports/test-report.html" --self-contained-html'])"
```

Comment:
Where/how is this report going to be collected? This seems like it would just be stuffed somewhere on the build worker and not propagated in a way that makes the test results visible.

Also, is there upstream documentation that describes what is covered by `test_aggregation`?

Reply:
true , currently up stream  tests report is only of local debugging . it needs some design and implementation work if we want to provide visibility and use the upstream tests report generated .

couldn't find any documentation as such. Took a look at what pandas methods its targetd. We can definetly  change what tests to trigger.
https://github.com/pandas-dev/pandas/blob/v1.4.2/pandas/tests/test_aggregation.py

Reply:
On my system, `test_aggregation` only picks up a few tests and doesn't provide nearly enough coverage:
```
$ python -c 'import pandas; pandas.test(["-k", "test_aggregation"])'
running: pytest -k test_aggregation /home/chl/.miniconda3-x86_64/envs/sklearn-test/lib/python3.8/site-packages/pandas
================================================= test session starts ==================================================
platform linux -- Python 3.8.13, pytest-7.1.1, pluggy-1.0.0
rootdir: /home/chl
plugins: forked-1.3.0, xdist-2.3.0, hypothesis-6.29.3
collected 153374 items / 153365 deselected / 44 skipped / 9 selected        
```

Reply:
yes this is the sample run to trigger sub set of tests, there are the available tags to trigger the subsets but very few are available in this pkg and we can trigger by passing folder or the file name we want to trigger. can you please provide some criteria on picking the sub tests? 

Tags: 
@pytest.mark.network
@pytest.mark.arm_slow
@pytest.mark.slow

Folder:
https://github.com/pandas-dev/pandas/tree/main/pandas/tests
Files:
test_common.py